src/BuildScriptGenerator/DotNetCore/DotnetCoreBashBuildSnippetProperties.cs

```diff
@@ -32,6 +33,8 @@ public class DotNetCoreBashBuildSnippetProperties
 
         public bool ZipAllOutput { get; set; }
 
+        public string ZippedOutputFileName { get; set; } = FilePaths.CompressedOutputFileName;
```

Comment:
Zipped and Compressed...ideally it should probably say compressed everywhere...i kept as zipped for now and will create a separate pr for cleaning this up...also need to see how to not break existing customers with a change to the property name `zip_all_output`

Reply:
we can still support `zip_all_output` but mark it as deprecated.
then, we can add a Checker that warns when deprecated properties are used.

Reply:
Good point about using Checker here.